src/redist/targets/packaging/osx/clisdk/resources/zh-hant.lproj/welcome.html

```diff
@@ -10,17 +9,17 @@
         <h2>.NET SDK</h2>
         <p id="title">.NET 是開發平台，可用來建置命令列應用程式、微服務和新式網站。其為開放原始碼、跨平台且由 Microsoft 支援。希望您喜歡!</p>
     </div>
-    <div align="left" style="font-family: Helvetica">
+    <div align="靠左" style="font-family: Helvetica">
```

Comment:
@cristianosuzuki77 I don't think they should be translating the align value here as that's html code. Is there something I can do on my side to mark it to not be translated or do I need to file a loc issue?

Reply:
I fixed this couple of weeks agon via the loc question ticket. This changes were ready to come back here.
However, since there is a manual commit to this PR, OneLocBuild task will not continue updating it.
You'll have to close the PR and let OneLocBuild task open a new PR with the fix.

You can see this information in the OneLocBuild task logs:
![image](https://user-images.githubusercontent.com/18152472/219483107-6aa1012e-93ce-4fb4-95bb-0acf27f849c2.png)